src/nidigital/system_tests/test_system_nidigital.py

```diff
@@ -294,8 +286,7 @@ def configure_for_history_ram_test(session):
     session.history_ram_pretrigger_samples = 0
     session.history_ram_number_of_samples_is_finite = True
 
-    session.burst_pattern(
-        site_list='site1',
+    session.sites[1].burst_pattern(
```

Comment:
Tests that use `fetch_history_ram_cycle_information` are disabled. Can you please update `fetch_history_ram_cycle_information` to accept site parameter as int.

Reply:
Done